micromamba/tests/test_env.py

```diff
@@ -120,7 +120,7 @@ def test_env_export(self):
         create("", "-n", env_name, "-f", spec_file)
         ret = yaml.safe_load(run_env("export", "-n", env_name))
         assert ret["name"] == env_name
-        assert set(ret["channels"]) == {"https://conda.anaconda.org/conda-forge"}
+        assert set(ret["channels"]) == {"conda-forge"}
```

Comment:
If we do add that the `defaults` naming, we should also add a test for it, so that we do not loose it through a refactoring.

Reply:
Indeed.